src/helpers/ProfileHelper/profileHelper.js

```diff
@@ -0,0 +1,79 @@
+import axios from 'axios';
+
+const profileHelper = (token) => axios.create({
+  baseURL: 'api/',
+  headers: { token }
+});
+
+const shortData = (date) => {
+  const length = 10;
+  const trimmedDate = date.substring(0, length);
+  return trimmedDate;
+};
+
+const componentDidMountHelper = () => {
```

Comment:
Why did you call it componentDidMountHelper?

Reply:
Because that function was called in componentDidMount function to validate some actions before componentDidMount function do an action.